http4s/src/main/scala/com/avast/grpc/jsonbrige/http4s/Http4s.scala

```diff
@@ -37,7 +37,10 @@ object Http4s extends StrictLogging {
               service.methodsNames.mkString("\n")
             }
 
-          case None => NotFound(s"Service '$serviceName' not found")
+          case None =>
+            val message = s"Attempt to GET non-existing service: $serviceName"
+            logger.warn(message)
```

Comment:
I'm not sure about so high log level. Imagine this will happen few thousand times when the service is found by some robot :wink: I propose `debug`.

Reply:
lowered to info